travel_platform/api/skyscanner/skyscanner_data_tests.py

```diff
@@ -0,0 +1,43 @@
+import unittest
+from skyscanner_data import *
+
+
+class SkyscannerDataTest(unittest.TestCase):
+    def setUp(self):
+        self.data = {"market": "UK", "currency": "GBP", "locale": "en-GB",
+                     "originplace": "SIN-sky", "destinationplace": "KUL-sky",
+                     "outbounddate": "2017-07-15", "inbounddate": "2017-07-16"}
+
+    def test_cheapest_quotes(self):
+        response = cheapest_quotes(self.data)
+        self.assertTrue(response)
+
+    def test_cheapest_quotes_if_params_in(self):
+        params = ['Carriers', 'Currencies', 'Places', 'Quotes']
+        response = True
+        for param in params:
+            if param not in cheapest_quotes(self.data):
```

Comment:
Did you have a chance to see how it was done by @kate-ka ?
There is much easier ways to compare if all the elements from list1 exists in list2.